cmake/common.cmake

```diff
@@ -1,3 +1,5 @@
+include("ggml/cmake/common.cmake")
```

Comment:
I think it would be better to have `llama.cpp`-specific flags instead of reusing the `ggml` flags like this. But this also works.

Reply:
I agree. I didn't know whether the re-use of `ggml_get_flags` was intentional or not, so I went with the least intrusive option. Well, it can easily be changed once the need arises.